docs/learning/tutorials/showcase-to-viewer.md

```diff
@@ -0,0 +1,141 @@
+# Adding showcase widgets to your iTwin Viewer
+
+### Prerequisites
+
+This tutorial assumes that you already have:
+
+- Explored the [sample showcase](https://www.itwinjs.org/sample-showcase).
+- Your own local source for the iTwin Web Viewer based on the template @bentley/itwin-viewer
+- Configured your local source to open the "House Model" sample iModel.
+  - Instructions to use this sample iModel can be found [here]($docs/learning/tutorials/create-test-imodel-sample/).
+- Understand the concept of a [UI Provider]($docs/learning/ui/abstract/uiitemsprovider) and adding [widgets]($docs/reference/ui-abstract/uiitemsprovider/uiitemsprovider/).
+- Completed ["Customizing the iTwin Viewer"]($docs/learning/tutorials/hello-world-viewer) tutorial.
+
+### Goal
+
+This tutorial will take widgets from the sample showcase and add them into your iTwin Viewer using the ```uiProviders``` prop.
+
+### Understanding the Sample Showcase
+
+We hope you have given the [sample showcase](https://www.itwinjs.org/sample-showcase) a tour and enjoyed the many samples we provide. You may want to use some of these samples in your own iTwin Viewer, and to do so you'll first need to understand how the showcase works.
+
+If we take a closer look at the files involved in each sample, you'll notice they all follow the same pattern containing a few important files:
+
+1. ```[SampleName]App.tsx``` - Corresponds to ```App.tsx``` in the itwin viewer template and provides the main ```Viewer``` component.
+2. ```[SampleName]Widget.tsx``` - Defines the ```UiItemsProvider``` that will be passed into prop ```uiProviders``` for our sample widget component. This widget is the controller for our samples.
+3. ```[SampleName]Api.ts``` - Defines widget functionality that uses the itwin.js API being showcased.
```

Comment:
itwin.js -> iTwin.js

Reply:
Added a new tile in the tutorials section for the entry.